packages/react-storage/src/components/StorageBrowser/views/LocationsView/getLocationsViewTableData/getLocationsViewTableData.ts

```diff
@@ -52,10 +51,11 @@ export const getLocationsViewTableData = ({
             return {
               key,
               type: 'text',
-              content: { text: getPermissionName(permission) },
+              content: { text: getPermissionName(permissions) },
             };
           }
           case 'action': {
+            // FIXME: conditional render download button if permissions include 'get'
```

Comment:
all these fixme should go out before launch right?

Reply:
This will be followed up separately.